src/js/profile/mobile/widget/Calendar.js

```diff
@@ -40,19 +40,23 @@
 				events = ns.event,
 				days = ["MON", "TUE", "WED", "THU", "FRI", "SAT", "SUN"],
 				fullNameMonth = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"],
-				today = new Date(),
-				todayYear = today.getFullYear(),
-				todayMonth = today.getMonth() + 1,
-				defaultToday = today.getUTCDate(),
+				dateData = new Date(),
+				todayYear = dateData.getFullYear(),
+				todayMonth = dateData.getMonth() + 1,
+				defaultToday = dateData.getUTCDate(),
+				loadTodayFlag = true,
+				activeMonth,
+				fixMonth,
 				switchElement,
+				arrowElement,
 				selectDay,
 
 				Calendar = function () {
 					this.options = utilsObject.merge({}, defaultOptions);
 				},
 
 				defaultOptions = {
-					pastSelection: false
+					pastSelection: "false"
```

Comment:
It have the option by "pastSelection".
The value that give whether date selection is possible or not, how about to use "pastSelection"?
I think "date-effect" is unintuitive.
How about to change the attribute name "data-effect"?

Reply:
"data-effect" is a data- * attribute.
The "effect" string can be changed.
data-effect is to specify the attribute in the html file.
pastSelection is the value of js applying that property.

Reply:
I agree @singa2000,
I think it would be good to sync the option with attribute name.
So the "data-past-selection" would be good to me.

Reply:
done.